src/openseadragon.js

```diff
@@ -206,6 +206,9 @@
   * @property {Number} [opacity=1]
   *     Opacity of the drawer (1=opaque, 0=transparent)
   *
+  * @property {String|Function} [placeholderFillStyle=null]
```

Comment:
I think it's worth calling out in the description what kind of string and function we're looking for (i.e. any string that's valid for the canvas fillStyle, or a function that returns same). Also, if we pass things into that function, that should be documented here.